### DISCUSSION TITLE
After deploy to prod: Uncaught ValueError: Path cannot be empty

### DISCUSSION THREAD

**Discussion by koma007** [symfony/symfony#50848] [Q&A]
After deploy on a production Apache server I got an error: 

[02-Jul-2023 16:03:37 Europe/Warsaw] PHP Fatal error:  Uncaught ValueError: Path cannot be empty in /vendor/symfony/dependency-injection/Loader/XmlFileLoader.php:714
Stack trace:
#0 /vendor/symfony/dependency-injection/Loader/XmlFileLoader.php(714): file_put_contents()
#1 /vendor/symfony/dependency-injection/Loader/XmlFileLoader.php(685): Symfony\Component\DependencyInjection\Loader\XmlFileLoader->shouldEnableEntityLoader()
#2 /vendor/symfony/config/Util/XmlUtils.php(77): Symfony\Component\DependencyInjection\Loader\XmlFileLoader->validateSchema()
#3 /vendor/symfony/config/Util/XmlUtils.php(132): Symfony\Component\Config\Util\XmlUtils::parse()
#4 /vendor/symfony/dependency-injection/Loader/XmlFileLoader.php(404): Symfony\Component\Config\Util\XmlUtils::loadFile()
#5 /vendor/symfony/dependency-injection/Loader/XmlFileLoader.php(47): Symfony\Component\DependencyInjection\Loader\XmlFileLoader->parseFileToDOM()
#6 /vendor/doctrine/doctrine-bundle/DependencyInjection/DoctrineExtension.php(162): Symfony\Component\DependencyInjection\Loader\XmlFileLoader->load()
#7 /vendor/doctrine/doctrine-bundle/DependencyInjection/DoctrineExtension.php(99): Doctrine\Bundle\DoctrineBundle\DependencyInjection\DoctrineExtension->dbalLoad()
#8 /vendor/symfony/dependency-injection/Compiler/MergeExtensionConfigurationPass.php(73): Doctrine\Bundle\DoctrineBundle\DependencyInjection\DoctrineExtension->load()
#9 /vendor/symfony/http-kernel/DependencyInjection/MergeExtensionConfigurationPass.php(42): Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationPass->process()
#10 /vendor/symfony/dependency-injection/Compiler/Compiler.php(73): Symfony\Component\HttpKernel\DependencyInjection\MergeExtensionConfigurationPass->process()
#11 /vendor/symfony/dependency-injection/ContainerBuilder.php(721): Symfony\Component\DependencyInjection\Compiler\Compiler->compile()
#12 /vendor/symfony/http-kernel/Kernel.php(487): Symfony\Component\DependencyInjection\ContainerBuilder->compile()
#13 /vendor/symfony/http-kernel/Kernel.php(709): Symfony\Component\HttpKernel\Kernel->initializeContainer()
#14 /vendor/symfony/http-kernel/Kernel.php(172): Symfony\Component\HttpKernel\Kernel->preBoot()
#15 /vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle()
#16 /vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#17 /public/index.php(5): require_once('...')
#18 {main}
  thrown in /vendor/symfony/dependency-injection/Loader/XmlFileLoader.php on line 714

Symfony v 6.3
even, when I try to deploy fresh: symfony new my_project_directory --version="6.3.*" --webapp I have the same error.

All these appears after:
composer install --no-dev --optimize-autoloader
and
clear:cache.

Any ideas?

**Comment by Hyperalpha** [comment]
Hello,

After adding the php module xlmrpc and clearing the cache, the error disappears for me.
